src/common/options.h

```diff
@@ -27,9 +27,22 @@
 #ifndef __OPTIONS_H__
 #define __OPTIONS_H__
 
+#include <stdlib.h>
+
 #include "float_const.h"
 #include "logger.h"
 #include "tinymt64.h"
+#include <pthread.h>
+
+/* Data type used to hold information required by the RNG used for MCA */
+typedef struct mca_data {
```

Comment:
This is not specific to mca but to random number generation. 
Please choose a name that reflects this such as `rng_state` instead.

Reply:
Done.